client/my-sites/plugins/not-available-error.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import EmptyContent from 'components/empty-content';
+import DocumentHead from 'components/data/document-head';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+function NotAvailableError( { translate } ) {
+	return (
+			<Main>
+				<DocumentHead title={ translate( 'Plugins' ) } />
+				<SidebarNavigation />
+				<EmptyContent
+					title={ translate( 'Not Available' ) }
+					line= { translate( 'The page you requested could not be found' ) }
```

Comment:
Why address this in a new pull request, @jorgefilipecosta? You're introducing the components with the error messages in this PR, so maybe it would make sense to have them addressed here?